src/main/java/com/codeborne/selenide/LocalStorageConditions.java

```diff
@@ -0,0 +1,15 @@
+package com.codeborne.selenide;
+
+import java.util.function.Predicate;
+
+public class LocalStorageConditions {
+
+  public static Predicate<LocalStorage> itemPresent(String item) {
+    return localStorage -> localStorage.containsItem(item);
+  }
+
+  public static Predicate<LocalStorage> itemHasValue(String item, String value) {
```

Comment:
For the sake of readability, it'd better rename it to `itemWithValue()` so that the user's code would read like this:

`localStorage().shouldHave(itemWithValue("mouse", "Jerry"),"Item 'mouse' value doesn't match")`

Reply:
not sure. We have only `should(predicate)` method method,` so should(itemWithValue)` looks not pretty)

Reply:
If we can only utilize "should", then I'm suggesting `should(containItemWithValue(...))` or something like that. It will also look good next to the suggested above `should(containItem(...))`.

Reply:
Thanks) looks nice) 
done)